analyzers/src/SonarAnalyzer.Common/SymbolicExecution/Roslyn/OperationProcessors/BranchingProcessor.cs

```diff
@@ -25,7 +25,7 @@ namespace SonarAnalyzer.SymbolicExecution.Roslyn.OperationProcessors;
 /// <summary>
 /// Base class for operation processors - used when operation takes a branching decision.
 /// See <see cref="SimpleProcessor{T}"/> if you need to return a single ProgramStates.
-/// See <see cref="BranchingProcessor{T}"/> if you just need to return multiple ProgramStates.
+/// See <see cref="MultiProcessor{T}"/> if you just need to return multiple ProgramStates.
```

Comment:
This should be reverted. `MultiProcessor` is base class for `SimpleProcessor` or `BranchingProcessor`. One of those should be used.